recipe/meta.yaml

```diff
@@ -293,6 +293,7 @@ outputs:
         - jupyterlab_execute_time >=3.1.2
         - jupyterlab_iframe >=0.5.0
         - jupyterlab_widgets >=3.0.10
+        - lsdb >=0.6.0  # [not aarch64]
```

Comment:
Do we still need to exclude this for aarch64? Migrations were done yesterday so I might work now.